### PR TITLE
Main

### PR DESCRIPTION
**Pull request by LiZoMark841619**
First pull

### CODE REVIEW

**Review by LiZoMark841619 — COMMENTED** [review]
Hello Ria can you see this?

File: Hurricane Analysis/git_help.txt
Comment (kalemaria) [inline Hurricane Analysis/git_help.txt:1]: Thanks, this is useful, but I will move the file to another place. :)

File: Hurricane Analysis/Hurricane Analysis.ipynb
Comment (kalemaria) [inline Hurricane Analysis/Hurricane Analysis.ipynb:1]: Thank you your refactoring, Balazs, it is hard to read your additions here and compare to the old code, so I'll try to merge them first and review later.

**Review by kalemaria — APPROVED** [review]

File: Hurricane Analysis/git_help.txt
Comment (LiZoMark841619) [reply Hurricane Analysis/git_help.txt:1]: Hello, ok, that is fine. Now I am replying in VSC, it is intereseting, I have not met this so far. What I meant is that using comprehnsion list and dict is highly recommendend, that's all. Anyhow your code was usable with good style.

**Pull request merged**